### PR TITLE
Add recording of launch webinar

### PR DESCRIPTION
**Pull request by zchase**
This PR will add the recording of the launch event to the website. Still waiting on the recording link, but I think we can get a review in before the link is live on YouTube.

### CODE REVIEW

File: layouts/webinars/single.html [hunk 1]
```diff
@@ -98,6 +103,25 @@ <h3 class="text-orange">What you will learn:</h3>
...
+            {{ if .Params.gated }}
+                <div class="md:mx-8">
+            {{ else }}
+                <div>
+            {{ end }}
+                <h3 class="text-orange">Get Started with Pulumi:</h3>
+                <ul class="text-gray-600">
+                    <li><strong><a href="{{ relref . "/docs/get-started/aws" }}">Deploy an AWS S3 Bucket</a></strong> with a KMS key that enables server-side encryption on the S3 bucket using the KMS key.</li>
+                    <li><strong><a href="{{ relref . "/docs/get-started/azure" }}">Create an Azure Resource Group and Storage Account</a></strong>, then export the storage account’s connection string.</li>
+                    <li><strong><a href="{{ relref . "/docs/get-started/gcp" }}">Create a GCP Storage Bucket</a></strong> and apply labels to that bucket.</li>
```
Comment (justinvp) [inline layouts/webinars/single.html:117]:
```suggestion
                    <li><strong><a href="{{ relref . "/docs/get-started/gcp" }}">Create a Google Cloud Storage Bucket</a></strong> and apply labels to that bucket.</li>
```

File: layouts/webinars/single.html [hunk 2]
```diff
@@ -98,6 +103,25 @@ <h3 class="text-orange">What you will learn:</h3>
...
+    <section id="gettingStartedList" class="px-4">
+        <div class="container mx-auto max-w-6xl pb-16">
+            {{ if .Params.gated }}
+                <div class="md:mx-8">
+            {{ else }}
+                <div>
+            {{ end }}
+                <h3 class="text-orange">Get Started with Pulumi:</h3>
+                <ul class="text-gray-600">
+                    <li><strong><a href="{{ relref . "/docs/get-started/aws" }}">Deploy an AWS S3 Bucket</a></strong> with a KMS key that enables server-side encryption on the S3 bucket using the KMS key.</li>
```
Comment (justinvp) [inline layouts/webinars/single.html:115]:
```suggestion
                    <li><strong><a href="{{ relref . "/docs/get-started/aws" }}">Create an AWS S3 Bucket</a></strong> with a KMS key that enables server-side encryption on the S3 bucket using the KMS key.</li>
```

File: layouts/webinars/single.html [hunk 3]
```diff
@@ -98,6 +103,25 @@ <h3 class="text-orange">What you will learn:</h3>
...
 
+    <section id="gettingStartedList" class="px-4">
+        <div class="container mx-auto max-w-6xl pb-16">
+            {{ if .Params.gated }}
+                <div class="md:mx-8">
+            {{ else }}
+                <div>
+            {{ end }}
+                <h3 class="text-orange">Get Started with Pulumi:</h3>
+                <ul class="text-gray-600">
```
Comment (justinvp) [inline layouts/webinars/single.html:114]: You think this list will perform better than linking directly to /docs/get-started?

Comment (zchase) [reply layouts/webinars/single.html:114]: I wanted the CTA to have a more natural feel on the page. And I felt this list felt like a good fit for the page over a few big buttons.

File: layouts/webinars/single.html (hunk 2 above)
Comment (zchase) [reply layouts/webinars/single.html:115]: 👍

File: layouts/webinars/single.html (hunk 1 above)
Comment (zchase) [reply layouts/webinars/single.html:117]: 👍

**Pull request merged**